low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -93,7 +94,7 @@ def set_health_data(health_data: dict, status, description=None,
                 'is' if good_state else 'is not')
         if not recommendation:
             recommendation = 'None' if good_state\
```

Comment:
"NA" could be better than "None" for any sensor data.

Reply:
done.